model/document.js

```diff
@@ -133,7 +133,10 @@ module.exports = memoize(function (db) {
 				label: this.database.resolveTemplate(this.label, this.getTranslations(), { partial: true }),
 				abbr: this.abbr,
 				status: this.status,
-				overviewSection: this.overviewSection.toJSON()
+				overviewSection: this.overviewSection.toJSON(),
+				statusLog: this.statusLog.ordered.map(function (stautLogEntry) {
```

Comment:
Also this should not be retrieved by creating an reactive array instance, but more as here -> https://github.com/egovernment/eregistrations/blob/master/model/requirement-upload.js#L153-L161